JS.md

```diff
@@ -88,6 +88,49 @@ In the [future :unicorn:][future streaming], this function can be
 extended to accept a [stream](https://streams.spec.whatwg.org), thereby enabling
 asynchronous, background, streaming compilation.
 
+#### `WebAssembly.instantiate`
+
+The `instantiate` function is overloaded based on types of its arguments.
```

Comment:
Did we in some way want to indicate this isn't in browsers yet.
Otherwise, lgtm.